src/cmdclass.py

```diff
@@ -130,7 +130,7 @@ def run(self):
     cmds["build_ext"] = cmd_build_ext
 
     if "cx_Freeze" in sys.modules:  # cx_freeze enabled?
-        from cx_Freeze.dist import build_exe as _build_exe
+        from cx_Freeze.command.build_exe import BuildEXE as _build_exe
```

Comment:
Unless cx_Freeze is very difficult to use any but the latest version of, it probably makes sense to support a range of versions.

```suggestion
        try:
            from cx_Freeze.command.build_exe import BuildEXE as _build_exe
        except ImportError:  # cx_Freeze < SOME_VERSION
            from cx_Freeze.dist import build_exe as _build_exe
```

Reply:
Created a new PR with this suggestion.  Thanks @spaam for the initial PR.